server/athenian/api/internal/features/github/pull_request_metrics.py

```diff
@@ -66,17 +65,24 @@ def group_prs_by_lines(lines: Sequence[int], items: pd.DataFrame) -> List[np.nda
     return group_by_lines(lines, items["size"].values)
 
 
-def group_prs_by_participants(participants: List[PRParticipants],
+def group_prs_by_participants(participants: Sequence[PRParticipants],
                               items: pd.DataFrame,
                               ) -> List[np.ndarray]:
     """
     Group PRs by participants.
 
     The aggregation is OR. We don't support all kinds, see `PullRequestFacts`'s mutable fields.
+    An array with selected indexes in `items` if returned for every group in `participants`.
+    If `participants` is empty a single array selecting everything is returned.
     """
-    # if len(participants) == 1, we've already filtered in SQL so don't have to re-check
-    if len(participants) < 2 or items.empty:
+    # FIXME: participants here can also be List[Dict[PRParticipationKind, Set[str]]]
+
+    # if there are no participant groups also we select everything and don't re-check
+    if len(participants) < 2:
```

Comment:
i think this is more readable than the version with a single if and min(len(participants))

Reply:
We should review the rest of the groupers for PRs, releases, JIRA, etc. for the same bug (in the future).